crux-metrics/src/crux/metrics/dropwizard.clj

```diff
@@ -0,0 +1,11 @@
+(ns crux.metrics.dropwizard
+  (:require [crux.metrics.bus :as met-bus]
+            [metrics.gauges :as gauges]))
+
+(defn cx-met->metrics [reg !metrics]
+  (map (fn [[fn-sym func]]
+         (gauges/gauge-fn reg (str fn-sym) #(func !metrics)))
+       (ns-publics 'crux.metrics.gauges)))
```

Comment:
using `ns-publics` here means that anything that gets added to `crux.metrics.gauges` is expected to be a gauge, but without there being anything in that namespace to suggest that might be the case - it might be clear in this PR, but the next person to refactor that ns, extracting a fn, say, might not spot it.

simpler - let's have crux.metrics.gauges export an explicit map, `(def gauges {...})`, say, from gauge-key -> gauge-fn, which this namespace can then iterate through

Reply:
I did that originally, but I didn't like that i was wringing out data that should already be there. Although I do agree it can break easily.